tensorflow_probability/python/distributions/gaussian_process_regression_model.py

```diff
@@ -288,6 +288,7 @@ def unnormalized_log_posterior(amplitude, length_scale, noise_variance):
 
   # Now we can sample from the posterior predictive distribution at a new set
   # of index points.
+  index_points = np.linspace(-1., 1., 200)[..., np.newaxis]
```

Comment:
Below a numpy object is passed in to index_points in the gprm. Can you replace that with `index_points=index_points`